src/functions/indBallRank.jl

```diff
@@ -41,7 +41,11 @@ function prox!{T <: Real}(f::IndBallRank, x::AbstractArray{T,2}, y::AbstractArra
     for j = 1:size(x,2)
       y[i,j] = 0.0
       for k = 1:f.r
-        y[i,j] += svdobj[:U][i,k]*svdobj[:S][k]*svdobj[:Vt][j,k]
+          if VERSION > v"0.5"
```

Comment:
Ok, thanks! I guess this change is necessary since [this](https://github.com/JuliaLang/julia/pull/19974), how should put the version cut more specific?